### DISCUSSION TITLE
Can't delete a blob storage

### DISCUSSION THREAD

**Discussion by thobenayann** [vercel/community#6620] [Help] [answered]
### Summary

I have exceeded the blob storage limit and want to delete this storage space.
However, the settings indicate that all blobs must be deleted before deleting the storage space, but the blobs are no longer visible in the space, preventing me from deleting them.

### Error

When i go on the blob managing page, i see an Vercel error toast "An error has occurred while fetching your blobs."

### Steps to Reproduce

1. Create a blob storage
2. Upload more than 250Mb
3. Try to delete the blob storage
4. Be stuck and can't delete it

**Comment by amyegan** [comment, marked as answer]
Hey @thobenayann. Can you give it another try?  We just made a change that should allow you to view the list and delete blobs even after limits are exceeded. Please let me know whether or not it works for you

**Reply by thobenayann** [reply]
Nice, it works !
Thank you for your reactivity 😄
